src/python/test/test_nextflow.py

```diff
@@ -117,7 +116,53 @@ def test_prepare_inputs_large_file(self):
         inputs = prepare_inputs("./nextflow/schema1.json")
         self.assertEqual(len(inputs), 93)
 
-
+class TestDXBuildNextflowApplet(DXTestCaseBuildNextflowApps):
+
+    def test_dx_build_nextflow_set_default_metadata(self):
+        pipeline_name = "hello"
+        applet_dir = self.write_nextflow_applet_directory(pipeline_name, existing_nf_file_path=self.base_nextflow_nf)
+        applet_id = json.loads(run("dx build --nextflow --json " + applet_dir))["id"]
+        app = dxpy.describe(applet_id)
+        self.assertEqual(app['name'], pipeline_name)
+        self.assertEqual(app['title'], pipeline_name)
+        self.assertEqual(app['summary'], pipeline_name)
+
+    def test_dx_build_nextflow_with_abs_and_relative_path(self):
```

Comment:
I would suggest adding a test that applet_dir ends with a slash. Plz see my comment above :)